README.md

```diff
@@ -0,0 +1,94 @@
+![Framework Logo](data/logo/logo.svg)
+
+Perform complex profile binned maximum likelihood fits by exploiting state-of-the-art differential programming. 
+Computations are based on the tensorflow 2 library and scipy minimizers with multithreading support on CPU (FIXME: and GPU).
+Implemted approximations in the limit of large sample size to simplify intensive computations.
+
+## Setup
+
+CombineTF2 can be run within a comprehensive singularity (recommended) or in an environment set up by yourself. 
+It makes use of the [wums](https://pypi.org/project/wums) package for storing hdf5 files in compressed format.
+
+### In a python virtual environment
+The simplest is to make a python virtual environment. It depends on the python version you are working with (tested with 3.9.18).
+First, make a python version, e.g. in the combinetf2 base directory (On some machines you have to use `python3`):
+```bash
+python -m venv env
+```
+The activate it and install the necessary packages
+```bash
+source env/bin/activate
+pip install wums tensorflow numpy h5py hist scipy matplotlib mplhep seaborn pandas plotly kaleido
+```
+The packages `matplotlib`, `mplhep`, `seaborn`, `pandas`, `plotly`, and `kaleido` are only needed for the plotting scripts. 
```

Comment:
can add a ```pip install combinetf2[plotting]``` option to automatically get the right dependencies for this as well?

Reply:
OK, I can try to add optional dependencies in the pip install